test/unit/sql/json.test.js

```diff
@@ -123,19 +129,22 @@ if (current.dialect.supports.JSON) {
 
         test('json function', () => {
           expectsql(sql.handleSequelizeMethod(Sequelize.json('json(\'{"profile":{"name":"david"}}\')')), {
-            default: 'json(\'{"profile":{"name":"david"}}\')'
+            default: 'json(\'{"profile":{"name":"david"}}\')',
+            //mysql: '`json(\'{"profile":{"name":"david"}}\')`->>\'$.\''
```

Comment:
can remove this line if its same as default. Please do the same for other tests below with commented mysql assertion

Reply:
Done, thanks @jedwards1211 !